authentication/templates/authentication/base.html

```diff
@@ -33,46 +33,10 @@
 
             <!-- Navbar links -->
             <div class="collapse navbar-collapse" id="collapsibleNavbar">
-                <ul class="navbar-nav mr-auto">
```

Comment:
We should probably make this base template as similar as possible to the new one (emis/templates/base.html). Use bootstrap 5 etc.

Reply:
This is the logged out - base so it does not have access to any apps.